pkg/util/membutil/membutil.go

```diff
@@ -11,11 +13,11 @@ import (
 )
 
 func WeightOf(membership *trantorpbtypes.Membership, nodeIDs []t.NodeID) tt.VoteWeight {
-	w := tt.VoteWeight(0)
+	sum := big.NewInt(0)
 	for _, node := range maputil.GetValuesOf(membership.Nodes, nodeIDs) {
-		w += node.Weight
+		sum.Add(sum, node.Weight.BigInt())
 	}
-	return w
+	return tt.VoteWeight(sum.String())
```

Comment:
Minor thing, but using sum.Text(16) (or the best would be [sum.Text(62)](https://pkg.go.dev/math/big#Int.Text), we can compact to the max the bigInt representation in string to reduce the size of a stable checkpoint. Probably not something to worry so much, but again, worth noting a "6n" optimization in a membership slice (which is sent over the network).

Reply:
this is because 0-9 (and a-z A-Z) use each 8 bits, so we optimize if using more characters.